### ISSUE TITLE
[Landing Page] : Implement mobile responsiveness

### ISSUE THREAD

**Issue by Teek-tech** [badging/eventBadging#80]
Welcome onboard....
Here's an opportunity to contribute to our [Badging Website.](https://badging-website.netlify.app)

The landing page is not mobile friendly and does not look good on different devices screen size.
You are required to inspect the UI design [here](https://www.figma.com/file/7tiuJbwakgDu15mOjzKTDK/Badging-Website?node-id=2115%3A6635&t=yaPK6oov4kd7YU9u-0) and provide an updated version your work by cloning the main branch.

**Extra Modifications**

- Hide/Show all necessary features, i.e, navigation and search icon, as clearly seen in our [figma design](https://www.figma.com/file/7tiuJbwakgDu15mOjzKTDK/Badging-Website?node-id=2115%3A6635&t=yaPK6oov4kd7YU9u-0)
- Implement a carousel design for the Testimony Section of the landing page for mobile screen only.
- Add the search field in the footer section
- Enable clickable event for the FAQ items too, it should not be restricted to the arrow icon.

**Your final work must look like this**

![Mobile Grid](https://user-images.githubusercontent.com/19340486/220692905-87c10f12-e009-4b06-bd50-e134e1f1abed.png)

If you want this task to be assigned to you, kindly inform us via the comment section.
Thank you for your contribution!

**Comment by OscarMesh** [comment]
I would love to work on this issue. Anticipating a feedback

**Comment by Ruth-ikegah** [comment]
awesome @OscarMesh I just assigned it to you!

**Comment by OscarMesh** [comment]
Alright. Will begin ASAP

**Comment by Ruth-ikegah** [comment]
Hi @OscarMesh are you still available to work on this issue?

**Comment by OscarMesh** [comment]
Yes, will tidy up this week. Working in the footer. Had a lot of
engagements, apologies for the delay

**Comment by Ruth-ikegah** [comment]
okay no worries @OscarMesh

**Comment by OscarMesh** [comment]
> 

Please note that I have created a PR

**Comment by Areebah-snow** [comment]
@Ruth-ikegah Id like to work on this

**Comment by ibimina** [comment]
@kaxada can I work on this?

**Comment by kaxada** [comment]
Let me look into this and revert

**Comment by kaxada** [comment]
hello @ibimina are you still available for this issue?

**Comment by OscarMesh** [comment]
Apologies for the delay, I'm currently not available right now, you can
assign it to someone else

Regards
